site/2017/04/01/announcing-doctrine-4.rst

```diff
@@ -0,0 +1,97 @@
+Announcing Doctrine 4
+=====================
+
+It has been few months since we shed some light on the `future of Doctrine project <https://github.com/doctrine/doctrine2/issues/6211>`__
+and had an extensive insight into what `Guilherme <https://twitter.com/guilhermeblanco>`__ had been up to. Adding
+tens of megabytes of IRC logs of internal discussion we felt we owe you an update on where Doctrine is and where
+it's heading to:
+
+Doctrine 3 is no more!

```

Comment:
Should this be using title capitalization (like "Join Forces" below)? If so, "Doctrine 3 Is No More!"